src/test/java/it/gov/pagopa/payment/controller/BasePaymentControllerIntegrationTest.java

```diff
@@ -132,8 +140,12 @@ private TransactionResponse createTrxSuccess(TransactionCreationRequest trxReque
         return trxCreated;
     }
 
-    private void checkTransactionStored(TransactionResponse trxCreated) {
+    private void checkTransactionStored(TransactionResponse trxCreated) throws Exception {
         TransactionInProgress stored = checkIfStored(trxCreated.getId());
+        SyncTrxStatusDTO storedMapped = new TransactionInProgress2SyncTrxStatusMapper().transactionInProgressMapper(stored);
```

Comment:
store the mapper as an instance variable instead of creating it for each invocation!
Moreover, it could be more readable to have the mapper invocation inside the asserEquals itself, as done in row 150